.ci/scripts/dist_s1/compare_dist_s1_products.sh

```diff
@@ -0,0 +1,170 @@
+#
+#    if [[ "${output_file##*/}" == *.tif* ]]
+#    then
+#      # Have to be a bit careful here since DIST-GEN-STATUS is a substring of DIST-GEN-STATUS-ACQ, we want to avoid
+#      # mapping DIST-GEN-STATUS-ACQ to DIST-GEN-STATUS since DIST-GEN-STATUS-ACQ == *DIST-GEN-STATUS*
+#      for potential_product in DIST-GEN-STATUS-ACQ DIST-GEN-STATUS GEN-METRIC BROWSE DATE-FIRST DATE-LATEST N-DIST N-OBS
+#      do
+#        if [[ "$output_file" == *"${potential_product}.tif" ]]; then
+#          product=$potential_product
+#          break
+#        fi
+#      done
+#
+#      echo "product is $product"
+#
+#      # Parse the tile code from the filename
+#      IFS='_' read -ra ARR <<< "$output_file"
+#      tile_code=${ARR[3]}
+#
+#      echo "tile code is $tile_code"
+#
+#      for potential_file in $(find $EXPECTED_DIR -type f -name '*.tif*')
+#      do
+#          if [[ "$potential_file" == *"$tile_code"*"$product".tif ]]; then
+#              echo "expected file is $potential_file"
+#              expected_file=$potential_file
+#              break
+#          fi
+#      done
+#
+#      if [ ! -f "$expected_file" ]; then
+#        echo "No expected file found for product type $product in expected directory $EXPECTED_DIR"
+#        overall_status=1
+#        compare_result="FAIL"
+#        compare_output="FAILED"
+#      else
+#         # compare output and expected files
+#         echo "python3 dist_s1_compare.py $(basename -- ${expected_file}) ${output_file}"
+#         compare_output=$(python3 $SCRIPT_DIR/dist_comparison.py ${expected_file} $OUTPUT_DIR/${output_file})
+#         echo "$compare_output"
+#      fi
+#
+#      if [[ "$compare_output" != *"FAIL"* ]]; then
+#          echo "Product validation was successful for $output_file"
+#          compare_result="PASS"
+#      else
+#          echo "Failure: Some comparisons failed for $output_file"
+#          compare_result="FAIL"
+#          overall_status=2
+#      fi
+#
+#    else
+#      echo "Not comparing file ${output_file}"
+#      compare_result="SKIPPED"
+#    fi
+#
+#    # add html breaks to newlines
+#    compare_output=${compare_output//$'\n'/<br>$'\n'}
+#
+#    update_html_results_file "${compare_result}" "${output_file}" "${expected_file}" "${compare_output}"
+#done
+#
+# Temporary: Just compare product dirs since that's what the script uses. Will ask ADT for band comparisons
+
+for output_product in $(find $OUTPUT_DIR -maxdepth 1 -mindepth 1 -type d)
+do
+  compare_output="N/A"
+  compare_result="N/A"
+  expected_file="N/A"
+
+  echo "output_product $output_product"
+  output_product=$(basename -- "$output_product")
+
+  # Parse the tile code from the filename
+  IFS='_' read -ra ARR <<< "$output_product"
+  tile_code=${ARR[3]}
+
+  for potential_product in $(find $EXPECTED_DIR -maxdepth 1 -mindepth 1 -type d)
+  do
+    if [[ "$potential_product" == *"$tile_code"* ]]; then
+      echo "expected product is $potential_product"
+      expected_product=$potential_product
```

Comment:
I think this should be `expected_file` not `expected_product`. It's causing N/A to be propagated to the output HTML report:

<img width="960" alt="Screenshot 2025-03-05 at 11 08 46 AM" src="https://github.com/user-attachments/assets/b151fe3a-00ab-4853-9ebb-51e09e5ad608" />

Reply:
![image](https://github.com/user-attachments/assets/797d8185-9e22-4da6-b4e2-1b366adf3c02)